backend/infrahub/core/schema/definitions/core.py

```diff
@@ -799,6 +800,25 @@
                     "default_value": AccountRole.READ_ONLY.value,
                     "enum": AccountRole.available_types(),
                 },
+                {
+                    "name": "status",
+                    "kind": "Dropdown",
+                    "choices": [
+                        {
+                            "name": ACCOUNT_STATUS_ACTIVE,
+                            "label": "Active",
+                            "description": "Account is allowed to login",
+                            "color": "#52be80",
+                        },
+                        {
+                            "name": "inactive",
+                            "label": "Inactive",
+                            "description": "Account is not allowed to login",
+                            "color": "#e74c3c",
+                        },
+                    ],
+                    "default_value": ACCOUNT_STATUS_ACTIVE,
```

Comment:
I think we have some other models defined as Dropdown where the choices are based on an enum. Instead here we mix them a bit by having a constant `ACCOUNT_STATUS_ACTIVE`, a string "inactive". I'm not sure I like the other way as it's also a bit redundant, so this is more of a comment. Could be that we'd change these to an enum at a later stage if we have other status entries and want to match against "inactive" or the new ones.